### DISCUSSION TITLE
Using css variable instead of scss variable ?

### DISCUSSION THREAD

**Discussion by oliv37** [twbs/bootstrap#39325] [Ideas] [answered]
Is there any reason to use scss variable `$link-decoration` rather than css variable `--#{$prefix}-link-decoration` in `_reboot.scss` ?
https://github.com/twbs/bootstrap/blob/ce114117a8710016022c40e717cb1f3ec95a5d7f/scss/_reboot.scss#L246

Using css variable would give to the user the possibility to change link decoration by overriding css variable at runtime.
Currently `--bs-link-decoration` variable is never used in [`bootstrap.css`](https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.css)

**Comment by louismaximepiton** [comment, marked as answer]
It will be handled by #39098, as explained inside this PR, we're waiting for v5.4.0 to introduce this change.
